book/gl/toc.md

```diff
@@ -19,29 +19,29 @@ No canto diso edite toc_template.md
-* [Requisitos comúns](../../translation/gl/patterns/common-requirements.md) - O código común dun repositorio compartido non satisfai as necesidades de tódolos equipos do proxecto que desexan empregalo; isto resólvese mediante a aliñación de requisitos e a refactorización.
-* [Servizo vs. libraría](../../translation/gl/patterns/service-vs-library.md) - Os equipos DevOps poden ser reticentes a traballar máis alá dos límites do equipo, en bases de código comúns; posto que non está definido quen sería a persoa responsable de responder ante unha situación de inactividade do servizo. Con frecuencia, a solución pode ser implantar o mesmo servizo en contextos independentes con cadeas de escalada separadas, en caso de caída do servizo ou de factorización dunha gran cantidade de código compartido nunha libraría na que se poidan facer colaboracións.
-* [Soporte grupal](../../translation/gl/patterns/group-support.md) - Que acontece se un equipo ou unha persoa abandona un proxecto InnerSource? Para manter vivo o proxecto fórmase un novo grupo de colaboradores/as interesados/as.
-* [Toma de decisións transparente entre equipos mediante RFC](../../translation/gl/patterns/transparent-cross-team-decision-making-using-rfcs.md) - Os proxectos InnerSource que desexan lograr un alto índice de participación e tomar as mellores decisións posibles para tódalas persoas involucradas, necesitan atopar formas de crear sistemas participativos durante todo o ciclo de vida do software. A publicación de documentos internos con peticións de comentarios (RFC) permite manter debates desde o comezo do proceso de deseño. Tamén aumenta as posibilidades de idear solucións cun alto grao de compromiso por parte de tódalas partes implicadas.
-* [Trusted commiter](../../translation/gl/patterns/trusted-committer.md) - Moitos proxectos InnerSource recibirán valoracións constantemente ou requirirán a posta en marcha de novas funcionalidades e a corrección de erros por parte dos/as contribuidores/as. Nestas situacións, as persoas encargadas do mantemento do proxecto buscan formas de recoñecer e recompensar o traballo do/a contribuidor/a máis alá das contribucións individuais.
-* [Valoración de proxectos entre equipos](../../translation/gl/patterns/crossteam-project-valuation.md) - É difícil convencer acerca do valor dos proxectos InnerSource desenvolvidos entre equipos que non teñen un impacto directo nos ingresos da empresa. Velaquí unha forma de representar o seu proxecto baseada en datos, que articula o seu valor e amplifícao.
+* [Agradecemento aos/ás participantes](../../translation/gl/patterns/praise-participants.md) - tras unha contribución a un proxecto InnerSource, é importante agradecerlle ao/á colaborador/a polo seu tempo e esforzo. Este modelo brinda unha orientación que non só recoñece dun xeito eficaz a súa contribución, senón que tamén xera un maior compromiso deste/a e doutros/as contribuidores/as.
```

Comment:
This file cannot be changed directly. The list of patterns here is auto-generated from the patterns files. The summary that you see here is what you see in the "Patlet" section in the respective pattern.